keyboards/keychron/k8/rgb/optical_iso/info.json

```diff
@@ -0,0 +1,197 @@
+{
+    "keyboard_name": "Keychron K8",
+    "usb": {
+        "pid": "0xFE0F"
+    },
```

Comment:
```suggestion
    },
    "matrix_pins": {
        "input_pressed_state": 1,
    },
```

Reply:
Added to the json

Reply:
(it's further down in the file, sorted "alphabetically" :))